src/main/java/org/apache/cassandra/sidecar/MainModule.java

```diff
@@ -126,6 +130,14 @@ public Router vertxRouter(Vertx vertx)
         return router;
     }
 
+    @Override
+    protected void  configure()
+    {
+        // TODO: Make the output type configurable
+        bind(CDCReaderMonitor.class).to(CDCReaderMonitorLogger.class);
+        bind(Output.class).to(ConsoleOutput.class);
```

Comment:
can we remove this method and like other variables such as HttpServer/Vertx/VertxRequestHandler/Router/Configuration use @Provide to get the real value and the the out put can be configurable ;
that is my code;
 @Provides
    @Singleton
    public Output outPut(Configuration conf)
    {
        String outPutClass = conf.getOutPut();
        if (!outPutClass.contains("."))
            outPutClass = "org.apache.cassandra.sidecar.common.output." + outPutClass; // I move the out put to common dir
        Output output = FBUtilities.construct(outPutClass, "output");
        return output;
    }

Reply:
`Output` is an interface. Let me see whether there's a benefit to refactoring it like this.

Reply:
I'm not seeing an advantage to the request, @Maxwell-Guo.  The approach @tharanga took is pretty standard for a Guice binding.